h2integrate/storage/battery/pysam_battery.py

```diff
@@ -113,6 +117,21 @@ def setup(self):
         self.config = PySAMBatteryPerformanceModelConfig.from_dict(
             merge_shared_inputs(self.options["tech_config"]["model_inputs"], "performance")
         )
+
+        self.add_input(
+            "charge_rate",
```

Comment:
I would prefer to call this `battery_power_kw`, especially since we are mimicking the ATB here. I could see an exception if we need to use `charge_rate` to stay general for the control/system interactions.